requirements/default.txt

```diff
@@ -125,8 +125,12 @@ django-honeypot==0.5.0 \
     --hash=sha256:962304f21055f05c73c41f2506f5a94f5b941e831a203b68e54cb5016bdb8d5e
 
 # Use Jinja2 as the template engine in Django
-django-jinja==2.2.1 \
-    --hash=sha256:04c53ba90cbede97e6543b98291f1eec259e57dfa4e014a5fbf0291afcdae6a9
+# Code: https://github.com/niwinz/django-jinja
+# Changes: https://github.com/niwinz/django-jinja/blob/master/CHANGES.adoc
+# Docs: http://niwinz.github.io/django-jinja/latest/
```

Comment:
Again, thanks for the project details. 2.3.0 added Django 1.11 compatibility, along with a reverted experiment in loading Jinja2 templates inside Django templates.